src/sdl2/gfx/primitives.rs

```diff
@@ -57,7 +53,10 @@ impl ToColor for u32 {
 impl ToColor for isize {
```

Comment:
thanks for fixing this up. however, this seems... very weird to me. when would this be used?

consider instead removing, as it looks like nothing relies on it but please confirm

Reply:
The comment on the impl block suggests that users could write hex colors as integers. I kinda get why it panics if the integer is not a valid color, but u32 could have been used instead to let users know at compile time.

I chose to just fix the UB/bug and not change the intended behavior, as a non breaking change.

> it looks like nothing relies on it

It is public API, users coud rely on it.

Reply:
Sure. Either way, please create an MR

Reply:
opened #1473

Reply:
It should be `to_be_bytes` if this is to allow users to write `0xrrggbbaa` colors, which should be converted to `(0xrr, 0xgg, 0xbb, 0xaa)` by `as_rgba()`, independent of the target byte order.
What there was here before this change was not any of `to_{n,l,b}e_bytes`, while acting like `to_ne_bytes` in practice, the compiler was free to shuffle the bytes in any way it liked.

Reply:
When I tried it acted like `to_ne_bytes`, but the author of the linked issue had something different.